config/default/kustomization.yaml

```diff
@@ -25,18 +25,18 @@ resources:
 # [CERTMANAGER] To enable cert-manager, uncomment all sections with 'CERTMANAGER'. 'WEBHOOK' components are required.
 #- ../certmanager
 # [PROMETHEUS] To enable prometheus monitor, uncomment all sections with 'PROMETHEUS'.
-#- ../prometheus
+- ../prometheus
 # [METRICS] To enable the controller manager metrics service, uncomment the following line.
-#- metrics_service.yaml
+- metrics_service.yaml
 
 # Uncomment the patches line if you enable Metrics, and/or are using webhooks and cert-manager
-#patches:
+patches:
 # [METRICS] The following patch will enable the metrics endpoint. Ensure that you also protect this endpoint.
 # More info: https://book.kubebuilder.io/reference/metrics
 # If you want to expose the metric endpoint of your controller-manager uncomment the following line.
-#- path: manager_metrics_patch.yaml
-#  target:
-#    kind: Deployment
+- path: manager_metrics_patch.yaml
```

Comment:
instad of using this patch just toss --metrics-bind-address=:8080 (or whatever port you want in deployment). 
Today AKS does not secure metrics ports of its addonds (theres actually a thread about this) so all existing addons just have them exposed.